hudi-client/src/main/java/org/apache/hudi/metrics/HoodieMetrics.java

```diff
@@ -59,6 +61,7 @@ public HoodieMetrics(HoodieWriteConfig config, String tableName) {
       this.deltaCommitTimerName = getMetricsName("timer", HoodieTimeline.DELTA_COMMIT_ACTION);
       this.finalizeTimerName = getMetricsName("timer", "finalize");
       this.compactionTimerName = getMetricsName("timer", HoodieTimeline.COMPACTION_ACTION);
+      this.indexLookupTimerName = getMetricsName("timer", "indexLookupTimer");
```

Comment:
change "indexLookupTimer" to  "indexLookup" ?

Reply:
Sure